packages/prices-api/src/proposal/util.ts

```diff
@@ -1,30 +0,0 @@
-import type { Proposal, ProposalStatus } from './models'
```

Comment:
Is there a way for your tools to exclude projects? This project is an NPM package, so unused exported functions may still be used by consumers of said package. (I know I use them in curvemonitor 🤓 )